install/kubernetes/helm/istio-remote/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+name: istio-remote
+version: 0.8.0
+description: Helm chart for Istio remote components
```

Comment:
Would be great if you can explain why people need it in the chart description.  e.g. you will need this when your remote k8s cluster needs to join Istio from an existing cluster.

Reply:
@linsun Yes, at present, people will need this chart (or a helm template yaml) to connect a remote kubernetes cluster to the main Istio control plane.  Should I just expand the description?

Reply:
@linsun note I did clarify the description a bit - PTAL.